### ISSUE TITLE
Show datasets user is following on profile + dashboard

### ISSUE THREAD

**Issue by DavidLemayian** [CodeForAfricaLabs/ckanext-followy#1]
As a user, I would like to see the datasets I am following on my profile page and dashboard when I'm logged in.

On profile page:

![screen shot on sep 14th at 05-22 pm](https://user-images.githubusercontent.com/877919/30422593-01611ed4-9973-11e7-8a21-b5aa97bbaf59.png)

On dashboard:

![screen shot on sep 14th at 05-36 pm](https://user-images.githubusercontent.com/877919/30422679-40b6653a-9973-11e7-8310-3ade39104f87.png)

**Comment by ignatiusukwuoma** [comment]
I am interested in working on this issue

**Comment by DavidLemayian** [comment]
@shemogumbe ☝️

**Comment by ignatiusukwuoma** [comment]
A PR has been created for this issue @DavidLemayian

**Comment by ignatiusukwuoma** [comment]
I think this issue can be closed now since the solution has been merged @DavidLemayian. Can this extension be used on africaopendata.org so more users can get to use it?

**Comment by DavidLemayian** [comment]
@andela-iukwuoma Yeah, I wanna close it once we've implemented on openAFRICA. Have we added it to https://github.com/CodeForAfricaLabs/openAFRICA ?
